contracts/ERC20SimpleSwap.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.5.11;
```

Comment:
Maybe it is time to start working with pragma statements without ^. It is generally advised if you are working with code that is in production.